modules/lib/lib-project/src/test/resources/test/DeleteProjectHandlerTest.js

```diff
@@ -0,0 +1,46 @@
+var assert = require('/lib/xp/testing.js');
+var project = require('/lib/xp/project.js');
+
+exports.deleteProject = function () {
+    createProject();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createProject' was used before it was defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5502328)